articles/network-watcher/network-watcher-nsg-flow-logging-portal.md

```diff
@@ -90,6 +90,9 @@ NSG flow logging requires the **Microsoft.Insights** provider. To register the p
     | Resource group | Select **Use existing**, and then select **myResourceGroup** |
 
     The storage account may take around minute to create. Don't continue with remaining steps until the storage account is created. If you use an existing storage account instead of creating one, ensure you select a storage account that has **All networks** (default) selected for **Firewalls and virtual networks**, under the **SETTINGS** for the storage account.
+    
+> [!NOTE]
+> While Microsoft.Insight and Microsoft.Network providers are currently supported for Azure Storage, NSG Flow logs is still not fully onboarded. To enable NSG Flow logging, Allow All Networks must still be selected until this feature is fully onboarded. 
```

Comment:
Please change line 95  to:
> While Microsoft.Insight and Microsoft.Network providers are currently supported for Azure Storage, NSG Flow logs are still not fully onboarded. To enable NSG Flow logging, **All Networks** must still be selected until this feature is fully onboarded.

Reply:
@kumud done, I put "All Networks" in bold as well.